### PR TITLE
Helm Uninstall Context Menu

### PR DESCRIPTION
**Pull request by shreyu86**
* adds `uninstall` context menu to Helm release explorer. 
* have included an confirmation dialog to avoid any accidents. 
* added option to copy release name as only having `Uninstall` in the context menu could lead to accidentally clicking on that option only.
part of the feature request #748

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -585,6 +586,11 @@
                     "group": "0@2",
                     "when": "view == extension.vsKubernetesHelmRepoExplorer && viewItem =~ /vsKubernetes\\.((chart)|(chartversion))/i"
                 },
+                {
+                    "command": "extension.helmUninstall",
+                    "group": "3",
```
Comment (shreyu86) [inline package.json:591]: was unsure here so have added a new group, happy to follow the conventions for groups if new group is not appropriate.

File: src/helm.exec.ts [hunk 2]
```diff
@@ -274,6 +274,31 @@ export function helmGet(resourceNode?: ClusterExplorerNode) {
...
+export function helmUninstall(resourceNode?: ClusterExplorerNode) {
+    if (!resourceNode) {
+        return;
+    }
+    if (resourceNode.nodeType !== NODE_TYPES.helm.release) {
+        return;
+    }
+    const releaseName = resourceNode.releaseName;
+    logger.log("⎈⎈⎈ Uninstalling " + releaseName);
+    vscode.window.showWarningMessage(`This will uninstall ${releaseName}, this cannot be undone.`, 'Uninstall').then((opt) => {
```
Comment (lstocchi) [inline src/helm.exec.ts:286]: I would change the message... maybe `You are about to uninstall ${releaseName}. This action cannot be undone.`

**Comment by shreyu86** [comment]
@lstocchi thank you for your comments. I have incorporated your feedback. Also noticed I had a couple of linting issues so fixed those too.

File: src/helm.exec.ts (hunk 2 above)
Comment (shreyu86) [reply src/helm.exec.ts]: Thank you, improved the message.

**Comment by lstocchi** [comment]
Thanks @shreyu86. It looks good to me. I saw that you also want to add other features. Do you want to keep this PR opened and add more stuff here or this can be merged?

**Comment by shreyu86** [comment]
@lstocchi I am still experimenting with rollback. I think if it's possible this would be phase 1 and can be merged. I'll open other MR soon.

**Comment by lstocchi** [comment]
Ok, merging this then. Thanks again @shreyu86

**Comment by shreyu86** [comment]
Awesome! Thank you! Looking forward to additional context menus for helm 😊

**Pull request merged**